pkg/systemd/services.html

```diff
@@ -122,6 +122,23 @@
             {{/TemplateDescription}}
           </table>
         </div>
+        {{#NotMetConditions.length}}
+        <div class="list-group-item">
+          <table>
+            <tr>
+              <td>
+                <span>condition failed</span>
```

Comment:
> And shouldn't `condition` be capitalized?

You tell me. Originally I had with it capital `C` but then I realized that nothing is capitalized in that view nor is 'condition failed' in service tool.